scvi/data/_manager.py

```diff
@@ -87,7 +86,7 @@ def __init__(
         self.fields = fields or []
         self.validation_checks = validation_checks or AnnDataManagerValidationCheck()
         self._registry = {
-            _constants._SCVI_VERSION_KEY: scvi.__version__,
+            #      _constants._SCVI_VERSION_KEY: scvi.__version__,
```

Comment:
@lauradmartens, any specific reason why you commented this part out? At first glance, this is the reason why the CI currently fails.